### PR TITLE
Logarithms Cheat Sheet: Correct notation for the calculus portion

### LINKED ISSUE #2929: Logarithms Cheat Sheet: Notation is wrong
The logarithms cheatsheet has that f'(a^x) = a^x * ln(a) (or something close). This may be a mixup of notation.
The formula says that applying the derivative of some function f to a^x gives the derivative of a^x. What was probably meant is (a^x)' = a^x * ln(x). That is, the derivative of exponential function. In general,  f'(g(x)) != g'(x).

------
IA Page:  http://duck.co/ia/view/logarithms_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @poornagurram

### PR DESCRIPTION
**Pull request by jain-ruchi**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] Improvement
    - [x] Bug fix: **`Logarithms Cheat Sheet: Fix #2929`**
    - [x] Chore: Use unicode characters for super and subscripts

###### Description of changes
Correct derivative notation for the calculus portion of the logarithms cheat sheet

###### Which issues (if any) does this fix?

Fixes #2929  - Change notation from `f'(g(x))` to `d(f(x))/dx` to minimize ambiguity

###### People to notify (@poornagurram)

---

Instant Answer Page: https://duck.co/ia/view/logarithms_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @poornagurram

### CODE REVIEW

**Comment by daxtheduck** [comment]
@jain-ruchi Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
@jain-ruchi how about (for absolute clarity): `d(a^x)/dx` ? currently `d/dx a^x` could be read as `d/(dx a^x)`

**Comment by jain-ruchi** [comment]
@mintsoft I agree -- amended commit

File: share/goodie/cheat_sheets/json/logarithms.json [hunk 1]
```diff
@@ -26,24 +26,24 @@
...
             },
             {
-                "key": "f'(ln(x))",
+                "key": "d(ln(x))/dx",
                 "val": "1/x,x>0"
             },
             {
-                "key": "f'(log(x)base a)",
-                "val": "1/xlna,x>0"
+                "key": "d(log(x)base a)/dx",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/logarithms.json]: Maybe we can write this as `log_a(x)` ? maybe that would be clearer?

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/logarithms.json]: How about `logₐ(x)`?

File: share/goodie/cheat_sheets/json/logarithms.json [hunk 2]
```diff
@@ -26,24 +26,24 @@
     "sections": {
         "Calculus": [
             {
-                "key": "f'(a^x)",
+                "key": "d(a^x)/dx",
                 "val": "a^xln(a)"
             },
             {
-                "key": "f'(e^x)",
+                "key": "d(e^x)/dx",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/logarithms.json]: Could we use Unicode characters for superscript `x`?

**Comment by jain-ruchi** [comment]
@GuiltyDolphin @mintsoft updated all superscripts and subscripts to use unicode characters instead

**Comment by GuiltyDolphin** [comment]
@jain-ruchi @mintsoft I don't know about you, but I'm in favor of parentheses around functions `cos(x)` (mostly the trig functions). 

For example, `logₐtanx` looks a bit odd to me - `logₐ(tanx)` would be better, maybe even `logₐ(tan(x))` thoughts?

**Comment by mintsoft** [comment]
@GuiltyDolphin that's an option, although log should be  `| |` rather than `( )` as it's undefined if the input is <0

**Comment by GuiltyDolphin** [comment]
@mintsoft True - it might get a bit *complex* otherwise.

**Comment by mintsoft** [comment]
@GuiltyDolphin Unfortunately there's no Github reaction for *groan*

**Comment by jain-ruchi** [comment]
@GuiltyDolphin @mintsoft so...what was the final verdict?

**Comment by mintsoft** [comment]
@jain-ruchi Thanks LGTM!

File: share/goodie/cheat_sheets/json/logarithms.json [hunk 3]
```diff
@@ -26,73 +26,73 @@
...
+                "key": "d(log\u2090u)/dx",
+                "val": "(du/udx)(log\u2090e)"
             }
             
         ],
         "Trigonometry": [
             {
-                "key": "log(tan x) base b ",
-                "val": "log(sinx) base b - log(cosx) base b"
+                "key": "log\u2090tanx",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/logarithms.json]: We should update this to the log_a(tan x) form when possible.

Comment (mintsoft) [inline share/goodie/cheat_sheets/json/logarithms.json]: Well spotted!

Comment (jain-ruchi) [inline share/goodie/cheat_sheets/json/logarithms.json]: It was merged before I could update it

Comment (mintsoft) [inline share/goodie/cheat_sheets/json/logarithms.json]: @jain-ruchi Apologies about that, I missed this! You can do another PR to fix it if you want :+1:

**Comment by edgesince84** [comment]
hey @jain-ruchi thank you for your contribution to DuckDuckHack, we really appreciate it!  If there are any additional updates you wanted to make, it's never too late! 👍 

If you're interested in working on something else, we recently launched a new forum with the initiative of improving programming-related searches, so I hope you'll [join us](https://forum.duckduckhack.com/)!

There are a lot of important tasks up for grabs that we'd like your help with, so feel free to and [jump in](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53).
